spec/google_coordinates_test.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../lib/google_coordinates.rb'
+
+describe "google coordinates functionality" do
+  # better tests would mock the API call
+  # todo: add tests for failed API call
+  it "returns the correct coordinates for 'Washington, DC'" do
+    expect(GoogleCoordinates.location_info('Washington, DC')).to eq(['Washington, DC, USA', '38.9071923,-77.0368707'])
```

Comment:
This would appear to make calls to the Google from the test. You suggest mocking being something worthwhile. Why might this be important?